Content-Organization.md

```diff
@@ -61,29 +63,34 @@ The Egeria core repository contains the core Egeria functionality, and is organi
   * **[admin-services](open-metadata-implementation/admin-services)** - supports the configuration of the OMAG Server Platform.  This configuration determines which of the open metadata and governance services are active.
   * **[common-services](open-metadata-implementation/common-services)** - support modules that are reused by other services.
     * **[ffdc-services](open-metadata-implementation/common-services/ffdc-services)** - provides base classes and validation for First Failure Data Capture (FFDC).
-    * **[gaf-metadata-management](open-metadata-implementation/common-services/gaf-metadata-management)** - provides metadata management for the [Governance Action Framework (GAF)](open-metadata-implementation/frameworks/governance-action-framework) properties and APIs.
```

Comment:
Not an issue with the gradle pom, but this is a useful list of the affected modules. Can you verify that when built, each maven module either has packaging=pom if an aggregator, or packaging=jar, AND contains 3 jars for the a) source b) javadoc c) built jar - otherwise the new modules may not be releaseable. Hopefully should happen by default but worth checking. If neiher a jar or pom we should figure out how to handle on a case by case basis